plugins/web-worker-offloading/third-party/woocommerce.php

```diff
@@ -23,10 +23,15 @@
 function plwwo_woocommerce_configure( $configuration ): array {
 	$configuration = (array) $configuration;
 
+	$configuration['globalFns'][] = 'gtag'; // Allow calling from other Partytown scripts.
+
+	// Expose on the main tread. See <https://partytown.builder.io/forwarding-event>.
+	$configuration['forward'][] = 'dataLayer.push';
+	$configuration['forward'][] = 'gtag';
```

Comment:
The addition of this line here means that scripts in the main thread will now be able to call `gtag()` whereas previously it was only available to other scripts offloaded to a worker.

Reply:
I can confirm this works now 👍